### PR TITLE
(BIDS-2619) fix vali history tooltip

### PR DESCRIPTION
**Pull request by LuccaBitfly**
### <samp>🤖 Generated by Copilot at 90dd7c9</samp>

This pull request fixes various typos and formatting issues in the code and the user interface of the eth2-beaconchain-explorer. It improves the validator history, attestation and block status badges, and the mempool view table. It affects the files `handlers/validator.go`, `utils/format.go` and `handlers/mempoolView.go`.

### CODE REVIEW

File: utils/format.go [hunk 1]
```diff
@@ -73,13 +73,13 @@ func FormatAttestationStatus(status uint64) template.HTML {
 // FormatAttestationStatusShort will return a user-friendly attestation for an attestation status number
 func FormatAttestationStatusShort(status uint64) template.HTML {
 	if status == 0 {
-		return `<span title="Scheduled" data-toggle="tooltip" class="mx-1 badge badge-pill bg-light text-dark" style="font-size: 12px; font-weight: 500;">Sche.</span>`
+		return `<span title="Scheduled Attestation" data-toggle="tooltip" class="mx-1 badge badge-pill bg-light text-dark" style="font-size: 12px; font-weight: 500;">Sche.</span>`
```
Comment (Eisei24) [inline utils/format.go:76]: **question:** I don't see it mentioned anywhere that this tooltip should be changed.
If this was discussed ignore this comment.

File: utils/format.go [hunk 2]
```diff
@@ -468,15 +468,15 @@ func FormatBlockStatus(status, slot uint64) template.HTML {
 func FormatBlockStatusShort(status, slot uint64) template.HTML {
 	// genesis <span class="badge text-dark" style="background: rgba(179, 159, 70, 0.8) none repeat scroll 0% 0%;">Genesis</span>
 	if status == 0 && SlotToTime(slot).Before(time.Now().Add(time.Minute*-1)) {
-		return `<span title="Scheduled" data-toggle="tooltip" class="mx-1 badge badge-pill bg-light text-dark" style="font-size: 12px; font-weight: 500;">Miss.</span>`
+		return `<span title="Scheduled Slot" data-toggle="tooltip" class="mx-1 badge badge-pill bg-light text-dark" style="font-size: 12px; font-weight: 500;">Miss.</span>`
```
Comment (Eisei24) [inline utils/format.go:471]: **question:** Like above I don't see it mentioned anywhere that this tooltip should be changed.
If this was discussed ignore this comment.

**Review by Eisei24 — COMMENTED** [review]
lgtm but please confirm that the tooltips should be changed.

File: utils/format.go (hunk 2 above)
Comment (LuccaBitfly) [reply utils/format.go]: After discussion, should be "Block" rather than "Slot". Thx and ptal

File: utils/format.go (hunk 1 above)
Comment (LuccaBitfly) [reply utils/format.go:76]: It was discussed and approved :)

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**